src/crypto/crypto_tls.cc

```diff
@@ -809,24 +746,62 @@ void TLSWrap::ClearOut() {
   // See node#1642 and SSL_read(3SSL) for details.
   if (read <= 0) {
     HandleScope handle_scope(env()->isolate());
-    int err;
+    Local<Value> exception;
```

Comment:
Can we call this `error`? It’s not really an exception unless it’s `throw`n :)

Reply:
Yea, that makes sense, updated :)